content/docs/releases/README.md

```diff
@@ -16,21 +16,18 @@ and other world events.
 <a id="supported-releases"></a>
 ## Currently supported releases
 
-| Release  | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
-|----------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-| [1.13][] | Sep 12, 2023 |    Release of 1.15     |            1.23 → 1.28             |           4.10 → 4.15             |
-| [1.12][] | May 19, 2023 |    Release of 1.14*    |            1.22 → 1.27             |            4.9 → 4.14             |
+|   Release    | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
+|--------------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
+|     [1.13][] | Sep 12, 2023 |    Release of 1.15     |            1.23 → 1.28             |           4.10 → 4.15             |
+| [1.12 LTS][] | May 19, 2023 |      May 19, 2025      |            1.22 → 1.28             |            4.9 → 4.15             |
 
-\*This is an LTS release sponsored by [Venafi](https://www.venafi.com/).
-It will be supported for a longer period of time than other releases.
-The EOL date for this release has not yet been determined & communicated, but can be decided by the Sponsor.
-The cert-manager maintainers guarantee that this release will be supported until the release of 1.14.
+cert-manager 1.12 is a Long Term Support (LTS) release sponsored by [Venafi](https://www.venafi.com/). It will continue to be supported for at least 2 years from release.
 
 ## Upcoming releases
 
 | Release  | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |----------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-| [1.14][] | Jan 15, 2024 | ~4 months post release |               TBD                  |                TBD                |
+| [1.14][] | Jan 15, 2024 | ~4 months post release |             1.24 → 1.29            |           4.11 → 4.15             |
```

Comment:
note: I left "4.15" as the latest OpenShift we'll support because even 4.14 only released a month ago and 4.15 isn't released yet. 4.16 seems to be so long away that it doesn't make sense to list it here yet.